docs/Accessibility.md

```diff
@@ -23,32 +23,52 @@ If you want to allow trips and stops of unknown wheelchair-accessibility then ad
 {
   "routingDefaults": {
     "wheelchairAccessibility": {
-      "trips": {
+      "trip": {
         "onlyConsiderAccessible": false,
         "unknownCost": 600,
         "inaccessibleCost": 3600
       },
-      "stops": {
+      "stop": {
         "onlyConsiderAccessible": false,
         "unknownCost": 600,
         "inaccessibleCost": 3600
-      }
+      },
+      "inaccessibleStreetReluctance": 25,
+      "maxSlope": 0.08333,
+      "slopeExceededReluctance": 50,
+      "stairsReluctance": 25
     }
   },
   "updaters": []
 }
 ```
 
-The parameters for `stops` and `trips` mean the following:
-
-| name                     |                                                                                      | default |
-|--------------------------|--------------------------------------------------------------------------------------|---------|
-| `onlyConsiderAccessible` | Whether to include unknown accessibility and inaccessible stops/trips in the search. | `false` |
-| `unknownCost`            | The cost to add if a stop/trip has unknown wheelchair accessibility                  | 600     |
-| `inaccessibleCost`       | The cost to add if a stop/trip is known to be inaccessible                           | 3600    |
-
-**Note**: Unless your accessibility data coverage is complete you will receive much better results by setting `onlyConsiderAccessible=true`, because
-otherwise you receive barely any results.
+The parameters for `stop`, `trip` and `elevator` mean the following:
+
+| name                     |                                                                                                | default |
+|--------------------------|------------------------------------------------------------------------------------------------|---------|
+| `onlyConsiderAccessible` | Whether to include unknown accessibility and inaccessible stops/trips/elevators in the search. | `false` |
```

Comment:
The description and the parameter name are the opposites.

Reply:
Yes, that was confusing. I changed that in my latest commit.